requirements_all.txt

```diff
@@ -34,6 +34,9 @@ blinkstick==1.1.7
 # homeassistant.components.sensor.bitcoin
 blockchain==1.3.1
 
+# homeassistant.components.thermostat.eq3btsmart
+bluepy>=1.0.0
```

Comment:
It seems that Python 3 compatibility was only released with v1.0.4: https://github.com/IanHarvey/bluepy/issues/122#issuecomment-203855438